spec/calculator_spec.rb

```diff
@@ -1 +1,72 @@
-# Your code here
\ No newline at end of file
+# Your code here
+require 'calculator.rb'
+
+describe Calculator do
+
```

Comment:
create a common instance for `Calculator` and use it throughout the specs.
`let(:c) { Calculator.new }`